scripts/discourse-sync/main.py

```diff
@@ -1,26 +1,31 @@
 import os
 import sys
 import yaml
-from pydiscourse import DiscourseClient
+from client import Client
 from pydiscourse.exceptions import DiscourseClientError
 
 
+# Staff notice to be added to every autogenerated doc
+STAFF_NOTICE = ("This doc was automatically generated from Juju code. Please "
+"do not try to edit it - instead, report any problems by leaving a reply "
+"below, and the Juju team will try to fix them as soon as possible.")
+
```

Comment:
The problem is that currently, none of the generated Markdown docs retain information about where the "source" is (even in the case when this concept makes sense). E.g. once you've run `juju documentation` and have generated the command docs, each doc doesn't contain information about where the original file is.

We would have to significantly change the whole workflow to enable this, and even then I'm not sure if it's worth the effort.

Some acceptable alternatives might be:
- just a generic link to the GitHub, not pointing to a specific file
- I could write a Markdown doc that lives in the code, and points to where the information is sourced from for each doc. This would at least give anyone who wants to open a PR a starting point on where to look.